misaligned.c

```diff
@@ -1,21 +1,49 @@
 #include <stdio.h>
 #include <assert.h>
 
-int printColorMap() {
-    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
-    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
-    int i = 0, j = 0;
-    for(i = 0; i < 5; i++) {
-        for(j = 0; j < 5; j++) {
-            printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
-        }
-    }
-    return i * j;
+const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+
+int ValidateColorFromPair(int index, const char* maj,const char* min);
+
+int printColorMap() 
+{
+	int i = 0, j = 0;
+	for(i = 0; i < 5; i++) 
+	{
+		for(j = 0; j < 5; j++) 
+		{
+			printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[j]);

```

Comment:
zero based color pair is printed... is this expected from a user point of view? or the color pair number needs to start from 1?
Also if that is the case, is there a test case to confirm that expectation?